src/test/java/picard/sam/markduplicates/MarkDuplicateWithMissingBarcodeTest.java

```diff
@@ -19,13 +19,20 @@ protected AbstractMarkDuplicatesCommandLineProgramTester getTester() {
         @Override
         public void runTest() {
             boolean hasRX = false;
+            boolean isDuplex = false;
             for (final String argument : this.getArgs()) {
                 if (argument.startsWith(getArgumentName())) {
                     hasRX = true;
                     break;
                 }
+                if (argument.startsWith("DUPLEX_UMI")) {
+                    isDuplex = true;
+                    break;
+                }
             }
-            if (!hasRX) addArg(getArgumentName() + "=" + getTagValue());
+            
+            // Tests that specify the RX tag, or the DUPLEX_UMI argument should not be used with missing barcodes.
+            if (!hasRX && !isDuplex) addArg(getArgumentName() + "=" + getTagValue());
```

Comment:
add braces

Reply:
@yfarjoun 
Back to you.  Was this the only request?